rosdep/python.yaml

```diff
@@ -4629,6 +4640,22 @@ python3-gitlab:
     xenial:
       pip:
         packages: [python-gitlab]
+python3-imaging:
```

Comment:
Our default is to use the debian package name for the key so I'd suggest we use python3-pil here as the key to be clear. python3-imaging is the old name and since this is a new key we might as well pick the newer name.

Reply:
Ok, switched. ```pil``` is also the old name. As you can see most distros use pillow in the name to differentiate the newer fork from the older project. Maybe ```python3-pillow```?